src/catchup/VerifyLedgerChainWork.cpp

```diff
@@ -197,56 +229,98 @@ VerifyLedgerChainWork::verifyHistoryOfSingleCheckpoint()
         return HistoryManager::VERIFY_STATUS_ERR_MISSING_ENTRIES;
     }
 
-    auto status = HistoryManager::VERIFY_STATUS_OK;
     if (curr.header.ledgerSeq == mRange.last())
     {
-        CLOG(INFO, "History") << "Verifying catchup candidate "
-                              << curr.header.ledgerSeq << " with LedgerManager";
-        status = mApp.getLedgerManager().verifyCatchupCandidate(curr,
-                                                                mManualCatchup);
+        // Verifying the most recent checkpoint means no other checkpoints are
```

Comment:
I think that we should refactor those two ifs: there is no reason to distinguish the "trusted ledger from SCP" and the "trusted ledger from the next checkpoint":
I would just set `nextCheckpointFirstLedger` to either the trusted value from SCP (if we're verifying the last checkpoint) or to the value of `mVerifiedAhead`

Reply:
Done.